build/xtask/src/dist.rs

```diff
@@ -122,6 +120,14 @@ impl PackageConfig {
         })
     }
 
+    fn img_dir(&self, img_name: &str) -> PathBuf {
+        self.dist_dir.join(img_name)
+    }
+
+    fn img_file(&self, name: impl AsRef<Path>, img_name: &str) -> PathBuf {
+        self.img_dir(img_name).join(name)
```

Comment:
Is this going to break Humility?

Reply:
It will not! Right now we generate a separate build.zip for each image so from humilty's perspective it's just loading another archive. Cliff and I briefly chatted about having a single build.zip but that's a longer term TODO and would require a humility bump.

Reply:
👍🏻